compute_endpoint/tests/unit/test_config_utils.py

```diff
@@ -33,7 +33,12 @@ def use_fs(fs):
     yield
 
 
-def test_render_user_config_escape_strings():
+@pytest.fixture
+def conf():
```

Comment:
A bare ``conf`` isn't as obvious as to why this is needed as a fixture or how it's different from a normal Config().  What do you think of naming it test_conf, mock_conf, or simple_conf?  And put that comment in the PR as a # line in the fixture ie. ``# empty executor list avoids unnecessary network call``

Reply:
Good suggestion.  Hang tight.